Lib/ensurepip/__init__.py

```diff
@@ -105,7 +91,7 @@ def version():
     """
     Returns a string specifying the bundled version of pip.
     """
-    return _get_packages()['pip'].version
+    return _get_pip_version()
```

Comment:
This can return the constant that is used to locate the pip wheel.

```suggestion
    return _PIP_VERSION
```